gapil/template/arith.go

```diff
@@ -0,0 +1,23 @@
+// Copyright (C) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package template
+
+func (*Functions) Add(x int, y int) int {
```

Comment:
Godocs please. Ubernit, but you can also share the types in parameters: `Add(x, y int)`

Reply:
Removed this file, as I realized I am no longer using it.